src/q5/network-INGI1341/summary/network-INGI1341-summary.tex

```diff
@@ -684,1820 +865,2049 @@ \subsubsection{Organisation to sharing bandwith}
+byte level). It also supplies information about the out-of-sequence chunks. Some
+other differences include:
 \begin{enumerate}
-    \item Il peut fournir de l'information sur \textbf{LES} différents ``trou'' dans le buffer de réception
-    \item Il peut donner un feedback sur chunck dupplicate (Annonce une mauvaise heuristic du sender)
+    \item Can supply information about the various ``holes'' in the reception
+    buffer.
+    \item Can give feedback about a duplicate chunk. (Points to a bad heuristic
+    on the sender's behalf.)
 \end{enumerate}
 
 \subsubsection{Connection release}
-Applique un three-way handshake pour cloturer la connection, ou un ABORT chunk
-pour refuser une connection ou la cloturer immédiatement.
+Uses a three-way handshake to close the connection, or an \texttt{ABORT} chunk
+to either refuse connection or close it immediately.
 
 \subsection{UDP - TCP - SCTP}
-\begin{table}[h]
+\begin{table}[ht]
     \begin{tabular}{|c|c|c|c|c|c|}
     	   \hline
-    	    Protocol& Reliablility&\#options&Mode&Establishment&Num seq\\
+    	    Protocol& Reliablility&\# options&Mode&Establishment&Num seq\\
         \hline
-        UDP & Unreliable   & x              & Stream-mode     & x                   & x \\
+        UDP & Unreliable   & x              & Stream-mode     &
+        x                   & x \\
         \hline
-        TCP & Reliable     & Option finie   & Stream-mode     & Three-way handshake & seq. = byte \\
-            &              &                &  Single host    &                     &  \\
+        TCP & Reliable     & Option finie   & Stream-mode     & Three-way
+        handshake & seq. = byte \\
+            &              &                &  Single host
+            &                     &  \\
         \hline
-        SCTP & reliable or & Option infinie & message mode as & Four-way handshake & chunk \\
-             & partially   & with chunk     & a stream mode   &                    & \\
-             &             &                & Multihomed host &                    & \\
+        SCTP & Reliable or & Option infinie & Message mode as & Four-way
+        handshake & chunk \\
+             & partially   & with chunk     & a stream mode
+             &                    & \\
+             &             &                & Multihomed host
+             &                    & \\
         \hline
     \end{tabular}
-    \caption{Difference between UDP, TCP and SCTP}
+    \caption{Differences between UDP, TCP and SCTP.}
 \end{table}
 \subsection{Congestion control}
-Le control est effectué dans la transport layer.
+The transport layer is where control happens.
 
-\textsc{tcp} control la congestion en agissant sur la window size puisque une connection
-ne peut pas envoyer des données plus vite que  $$\frac{window}{rtt}\quad window=min(swin,rwin)$$
+TCP controls congestion by acting on the window size since a
+connection can't send data faster than \[\frac{\textnormal{window}}{rtt}\quad
+\t
```

Comment:
Line 2566:
Could be good to say that in this summary we use interchangeably the terms "switch" and "bridge", although the latter term is more general.